src/Compilers/Server/ServerShared/ServerDispatcher.cs

```diff
@@ -236,6 +236,8 @@ private void HandleCompletedGCTask()
         {
             _gcTask = null;
```

Comment:
While you're here, we might as well do a LOH compaction. Would you mind adding that request?